dom.bs

```diff
@@ -6767,6 +6767,22 @@ invoked, must run these steps:
  "<code>section</code>", or
  "<code>span</code>", then <a>throw</a> a "{{NotSupportedError!!exception}}" {{DOMException}}.
 
+ <li>
+  <p>If <a>context object</a>'s <a for=Element>local name</a> is a
```

Comment:
Need to change this to something like:

    If context object's local name is a valid custom element name, or
    context object's is value is not null, then:

Reply:
Fixed in a new commit.